cmd/root.go

```diff
@@ -95,12 +172,31 @@ func initConfig() {
 		}
 	}
 	if !viper.IsSet("host") {
-		viper.Set("host", constants.Host)
+		viper.Set("host", utils.Host)
 	}
 	if !viper.IsSet("auth_token") {
-		viper.Set("auth_token", constants.AuthToken)
+		viper.Set("auth_token", utils.AuthToken)
 	}
-	viper.WriteConfig()
+	if !viper.IsSet("consent_telemetry") {
```

Comment:
nit:

This variable looks bit odd to me., why not just `telemetry`?

Reply:
Just following RH standards, don't want to reinvent the wheel
https://github.com/code-ready/crc/blob/master/pkg/crc/segment/segment.go#L64